app/assets/stylesheets/adopt.css.scss

```diff
@@ -35,3 +36,58 @@
     font-weight: normal;
   }
 }
+
+ul.list-of-rats {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+  text-align: center;
+}
+
+ul.list-of-rats li{
+  display: inline-block;
+  height: 150px;
+  margin: 100px 1em 1em 0;
+  position: relative;
+  width: 150px;
+}
+
+span.text-content {
+  background: rgba(0,0,0,0.5);
+  color: $blue;
+  cursor: pointer;
+  display: table;
+  height: 100px;
+  left: 0;
+  position: absolute;
```

Comment:
Properties should be ordered background, color, cursor, display, height, left, opacity, position, top, -moz-transition, -o-transition, -webkit-transition, transition, width